eng/devices/ios.cake

```diff
@@ -93,7 +98,16 @@ void Cleanup()
 		Information("No simulators found to delete.");
 		return;
 	}
-	var xharness = sims.Where(s => s.Name.Contains("XHarness")).ToArray();
+	var simulatorName = "XHarness";
+	if(iosVersion.Contains("17"))
```

Comment:
I think we need to open a bug on. xharness about this it seems it doesn't have the "Created by XHarness" if it's a iPhone 15. this leads to sometimes delete existing simulators.